src/org/zaproxy/zap/extension/scripts/ExtensionScriptsUI.java

```diff
@@ -172,6 +183,24 @@ public boolean canUnload() {
     	return true;
     }
 	
+	@Override
+	public void optionsLoaded() {
+		// Run any enabled scripts
+		for (ScriptWrapper script : this.getExtScript().getScripts(extScriptType)) {
+			if (script.isEnabled()) {
+				// Run the install 
+				ExtenderScript ec;
+				try {
+					ec = extScript.getInterface(script, ExtenderScript.class);
+					ec.install(getExtensionScriptHelper());
+					this.installedExtenderScripts.add(script.getName());
+				} catch (Exception e) {
+					LOGGER.error("Failed to install extender script", e);

```

Comment:
Could be a warning? (Most likely not an error in ZAP but in the script.) It could also output in the Script Console the error (and mark the script with error, `ExtensionScript.setError`).